notesapi/v1/models.py

```diff
@@ -35,6 +35,6 @@ def create(cls, note_dict):
 
         note_dict['ranges'] = json.dumps(ranges)
         note_dict['user_id'] = note_dict.pop('user', None)
-        note_dict['tags'] = json.dumps(note_dict.get('tags', list()))
+        note_dict['tags'] = json.dumps(note_dict.get('tags', list()), ensure_ascii=False)
```

Comment:
I'm a little confused by this.  It sounds like we are using json.dumps to make a string, and then storing that string somewhere that doesn't interpret JSON?